src/components/Articles/Articles.jsx

```diff
@@ -0,0 +1,16 @@
+import React from "react"
+import ArticleList from "./ArticleList"
+import { Route, Routes } from "react-router-dom"
+
+const ArticlesPage = () => {
+  return (
+    <div className="articles">
+      <h2>Welcome to the Articles Page</h2>
+      <Routes>
+        <Route path="/" element={<ArticleList />} />
+      </Routes>
```

Comment:
I'm not sure you need to have routes here, you could just render your ArticleList underneath the h2 as your routes have already been handled in App.js for this page